lib/administrate/version.rb

```diff
@@ -1,3 +1,3 @@
 module Administrate
-  VERSION = "0.2.0".freeze
+  VERSION = "0.2.1".freeze
```

Comment:
Understood. We should probably move to a beta model then. Such as `0.3.0.beta` :)